internal/trigger/messagebus/messaging_test.go

```diff
@@ -41,7 +41,10 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-var logClient logger.LoggingClient
+// Note the constant TriggerTypeMessageBus can not be used due to cyclic imports
+const TriggerTypeMessageBus = "EDGEX-MESSAGEBUS"
```

Comment:
good comment to know, but in theory you may want to refactor that constant into another package so that it can break out the cyclic restriction; i am ok with this one since it is just a test here

Reply:
I am hoping my upcoming refactoring for interfaces will resolve the cyclic import issues.